.travis.yml

```diff
@@ -26,28 +26,32 @@ script:
 - docker tag todo-app:prod legendarypan/todo-app:prod
 - docker push legendarypan/todo-app:prod
 
-# Pre-deploy Install
-- curl -sS https://cli-assets.heroku.com/install.sh | sh
+# Pre-deploy Install - removed post module 11
+# - curl -sS https://cli-assets.heroku.com/install.sh | sh
 # - heroku update
 # - heroku plugins:install @heroku-cli/plugin-container-registry
```

Comment:
Let's delete these rather than just commenting them out, we can recover them from git if we ever need to